skyportal/models.py

```diff
@@ -80,14 +82,20 @@ class Stream(Base):
 User.groups = relationship('Group', secondary='group_users',
                            back_populates='users')
 
+
 @property
 def token_groups(self):
     return self.created_by.groups
+
+
 Token.groups = token_groups
 
 
 class Source(Base):
     id = sa.Column(sa.String, primary_key=True)
+    is_source = sa.Column(sa.Boolean, default=False)
```

Comment:
For SkyPortal, should this be true by default?

Reply:
I think we'll have far more candidates than sources, right? Then it would make more sense to me that this is False by default.